src/app.ts

```diff
@@ -315,13 +315,17 @@ function supportsFancyCharacters(): boolean {
   // There is no sure-fire way, but we know that most problems are when running in the IG Publisher,
   // so try to detect that situation (which is still actually pretty tricky and not guaranteed).
 
-  // 1. Many JVM will insert an environment variable indicating the main Java class being run.
+  // 1. If the user wants the fancy characters, allow them
+  if (Object.keys(process.env).some(k => k === 'FORCE_FANCY_CHARACTERS')) {
+    return true;
+  }
```

Comment:
This returns true as long as the `FORCE_FANCY_CHARACTERS` environment variable exists. This means that even if someone sets the value to `false` or `0`, it would still force fancy characters. I think it would be better to explicitly look for the value to be `true` or `1`. Does that seem reasonable?

Reply:
Indeed, I've added a check on `!!process.env.FORCE_FANCY_CHARACTERS`.